os/android/iahwc2.cpp

```diff
@@ -1083,6 +1083,14 @@ int IAHWC2::HookDevOpen(const struct hw_module_t *module, const char *name,
   return 0;
 }
 
+hwcomposer::NativeDisplay *IAHWC2::GetDisplay() {
+  return extended_displays_.at(0).GetDisplay();
```

Comment:
Okay. I'll do it.
But why does extended_display_.at(0) also works for me on Joule?.  Does it mean  extended_display_.at(0) is primary_display_?.
Thanks